app/src/main/java/com/example/sam/bucketlist/models/UserFields.java

```diff
@@ -1,4 +1,4 @@
-package com.example.sam.bucketlist.Fields;
+package com.example.sam.bucketlist.models;
 
```

Comment:
The package structure in Android is majorly based either on task or type of files. You can have fragments, activities packages etc.
Take a look here to understand [CodePath](https://guides.codepath.com/android/Organizing-your-Source-Files) or [Hackernoon](https://hackernoon.com/package-by-features-not-layers-2d076df1964d)